setup/initial/aws-setup-app-server.sh

```diff
@@ -33,13 +33,16 @@ init_ec2_instance
 # Install Java 11 (Correto)
 amazon-linux-extras install -y java-openjdk11
 
-yum install -y python3
+# python3-devel, Development Tools and wheel packages are needed to build wheels including C/C++ code
+yum install -y python3 python3-devel
```

Comment:
maybe architecture check and install these dependencies only for ARM instances?

Reply:
Okay.

Reply:
Done.